promoted_python_delivery_client/client/serde.py

```diff
@@ -0,0 +1,61 @@
+import ujson
+from promoted_python_delivery_client.model.request import Request
+from promoted_python_delivery_client.model.response import Response
+from promoted_python_delivery_client.model.log_request import LogRequest
+
+
+delivery_request_schema = Request.schema()  # type: ignore this is from dataclass_json
+log_request_schema = LogRequest.schema()  # type: ignore this is from dataclass_json
+
+
+# DEPRECATED use delivery_request_to_json_3
+def delivery_request_to_json(req: Request) -> str:
+    return req.to_json()  # type: ignore this is from dataclass_json
+
+
+# DEPRECATED use delivery_request_to_json_3
```

Comment:
Interesting.  Why not remove them now?  Was this just for the performance test?

Reply:
Yes, I would rather keep them around, not just for that but also for the possibility that if a specific dataclasses-json issue gets fixed they might turn back into the correct approach.